sources/packages/backend/apps/api/src/services/application-bulk-withdrawal/application-bulk-withdrawal-import-text.service.ts

```diff
@@ -46,8 +45,6 @@ export class ApplicationWithdrawalImportTextService {
   readText(textContent: string): BulkWithdrawalFileData {
     const applicationWithdrawalModels: ApplicationWithdrawalImportTextModel[] =
       [];
-    // Remove BOM(Byte order mark), if present.
-    textContent = removeUTF8BOM(textContent);
```

Comment:
If the method `removeUTF8BOM` is no longer being used it can be completely removed from the code.